src/content/docs/alerts-applied-intelligence/new-relic-alerts/learn-alerts/alerts-concepts-workflow.mdx

```diff
@@ -39,7 +39,7 @@ To help you get started, here are some alerts concepts and the workflow you'll f
 
 ## Introduction to important concepts [#terminology]
 
-![New Relic Alerts workflow diagram](./images/new-relic-alert-creation-workflow-diagram.png "new-relic-alert-creation-workflow-diagram.png")
+![Alerts workflow diagram](./images/new-relic-alert-creation-workflow-diagram.png "new-relic-alert-creation-workflow-diagram.png")
```

Comment:
@akristen, while you're in the neighborhood, you might just make the image title match the alt-text. That current title with the image name isn't too helpful.

Reply:
@akristen, were you going to update the title? Here's what I was expecting: 

```
![Alerts workflow diagram](./images/new-relic-alert-creation-workflow-diagram.png "Alerts workflow diagram")
```